media-collection/TAGS_IMPL.md

```diff
@@ -0,0 +1,45 @@
+# Overview
+
+Customers that are using Adobe Tags to implement Adobe solutions, can implement Media using Web SDK Tags Extension.
+
+# Configuration
+
+To configure Media, when installing the Web SDK Extension, there is a Media section where settings like `player name`,
+`channel` have to be provided.
+![image](public/img/configuration.png)
+
+# Create a session
+
+To create a Media Session, similarly to the standalone library implementation, session details are needed as well as an
+alias to that media session, this can be a random string, the element ID of the player.
+![image](public/img/createSession.png)
+
+To receive the session ID, there is an event that triggers once the response from server is returned and has
+the sessionID in the `event.sessionId`. It can be stored to a Data Element and reused later using a Custom Code
+action provided by Core extension.
+![img.png](public/img/sessionReceived.png)
+Using Custom Code action it can be stored to a Data Element.
+
+```javascript
+_satellite.setVar("sessionId", event.sessionId);
+```
+
+# Send a Media Event
+
+After a Media Session has been successfully created, following Media Events can be triggered using the `Send Media Event` action.
+The action allows to handle it in two modes: automatic and non-automatic mode, which can be enabled
+checking `Handle Media Session Automatically`.
+Once the `eventType` is selected the appropriate form will be rendered, so that related details are provided to the event.
```

Comment:
I think we should add more explanation on the `player ID.` We don't want customers to believe this is the HTML ID attribute of the player tag necessarily, although it can be.

Reply:
Also, the `Quality of Experience data` gets hidden when you select `Handle Media Session Automatically`. That shouldn't happen right?